github/repositories/terraform-aws-modules/common.hcl

```diff
@@ -34,4 +34,87 @@ inputs = {
     }
 
   }
+
+  labels = {
+    "breaking change" = {
+      color       = "d66d0a"
+      description = "Breaking change features"
+    }
+
+    "bug" = {
+      color       = "d73a4a"
+      description = "Something isn't working"
+    }
+
+    "duplicate" = {
+      color       = "cfd3d7"
+      description = "This issue or pull request already exists"
+    }
+
+    "documentation" = {
+      color       = "0075ca"
+      description = "Improvements or additions to documentation"
+    }
+
+    "enhancement" = {
+      color       = "a2eeef"
+      description = "New feature or request"
+    }
+
+    "good first issue" = {
+      color       = "7057ff"
+      description = "Good for newcomers"
+    }
+
+    "help wanted" = {
+      color       = "008672"
+      description = "Extra attention is needed"
+    }
+
+    "invalid" = {
+      color       = "e4e669"
+      description = "This doesn't seem right"
+    }
+
+    "on hold" = {
+      color       = "f9d0c4"
+      description = "Needs further work or discussion"
+    }
+
+    "question" = {
+      color       = "d876e3"
+      description = "General question"
+    }
+
+    "wip" = {
+      color       = "a5f7a7"
+      description = "Work in progress"
+    }
+
+    "wontfix" = {
+      color       = "ffffff"
+      description = "This will not be worked on"
+    }
+
+    "needs triage" = {
```

Comment:
I think one additional label could be very useful - `needs example update` or something along those lines. we see it a lot where users will submit something but not update the example accordingly

Reply:
I would put this in generic way "needs input"

Reply:
`needs example update`, `needs docs update` - something user will have to make.

How about adding a new one - `needs examples/docs update`?

Reply:
It is not `input` as such but when a user opens a PR with just changes in the terraform code - it is sometimes not enough. It is not `input`, but rather `output`... "needs output"? :)

Reply:
ya, maybe combining with what @daroga0002 said - what about a generic `needs update`

Reply:
I think "needs input" should be ok, as in issue or PR tou can aleays specify directly what is required. Label will be usefull in filtering issues, but too many of them can be also problematic.

I think currently this set will be optimal.

Reply:
ok, `needs update`